src/components/PoolList/index.js

```diff
@@ -175,9 +174,8 @@ function PoolList({ pools, color, disbaleLinks, maxItems = 10, useTracked = fals
           <DataText area="name" fontWeight="500">
             {!below600 && <div style={{ marginRight: '20px', width: '10px' }}>{index}</div>}
             <TokenLogo
-              size={below600 ? 16 : 20}
+              size={below600 ? '16px' : '20px'}
```

Comment:
why change style here?

Reply:
size unit of parameter is necessary when it type string and the source code is like this